community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/slurmsync.py

```diff
@@ -470,6 +472,92 @@ def update_topology(lkp: util.Lookup) -> None:
         log.debug("Topology configuration updated. Reconfiguring Slurm.")
         util.scontrol_reconfigure(lkp)
 
+
+def delete_reservation(reservation_name: str) -> None:
+    util.run(f"{lkp.scontrol} delete reservation {reservation_name}")
+
+
+def create_reservation(reservation_name: str, node: str, starttime: str) -> None:
+    # Format time to be compatible with slurm reservation.
+    start_time=datetime.strptime(starttime, '%Y-%m-%dT%H:%M:%S%z')
+    formatted_start_time = start_time.strftime('%Y-%m-%dT%H:%M:%S')
+    util.run(f"{lkp.scontrol} create reservation user=slurm starttime={formatted_start_time} duration=180 nodes={node} reservationname={reservation_name}")
+
+
+def get_slurm_reservation_maintenance() -> Dict[str, str]:
```

Comment:
`reservation['start_time']['number']` is epoch `int`, please update type annotation. (optionally turn into `datetime`)

Reply:
Updated both to datetime object until we create reservation.